composer/src/composer.rs

```diff
@@ -1282,7 +1319,7 @@ impl ComposeTest {
     async fn create_container(
         &self,
         spec: &ContainerSpec,
-        ipv4: Ipv4Addr,
+        ipv4: Option<Ipv4Addr>,
```

Comment:
Why is the ip optional, because it's not in control of docker so it's up to the host?
Can we not set 127.0.0.1 in case of host?

Reply:
I see, makes sense